docker/development/docker-compose.yml

```diff
@@ -134,7 +134,32 @@ services:
       - redis
       - mailcatcher
 
+  nginx:
+    image: openresty/openresty:alpine
+    # Until https://github.com/nginx-proxy/docker-gen/pull/311 is merged, use hardcoded name
+    container_name: mampf-docker-nginx
+    ports:
+      - "127.0.0.1:3000:80"
+    volumes:
+      - "nginx-conf-d:/etc/nginx/conf.d"
+      - "submissions:/private/submissions:ro,nocopy"
+      - "public:/public"
+
+  dockergen:
+    build:
+      context: ./../..
+      dockerfile: docker/Dockerfile.dockergen
+    command: -notify-sighup mampf-docker-nginx -watch /etc/docker-gen/templates/nginx.tmpl /etc/nginx/conf.d/default.conf
+    restart: always
+    volumes:
+      - "nginx-conf-d:/etc/nginx/conf.d"
+      - "../nginx.tmpl:/etc/docker-gen/templates/nginx.tmpl"
```

Comment:
why `../nginx.tmpl` and not `docker/nginx.tmpl`?

Reply:
The working directory is such that the intuitive path docker/nginx.tmpl becomes ../nginx.tmpl internally.